training_scripts/gsgnn_dt/distill_debug.yaml

```diff
@@ -0,0 +1,65 @@
+---
+# TODO (HZ): add config example with and without GNN inference.
+version: 1.0
+gsf:
+  basic:
+    backend: gloo
+    ip_config: ip_list_debug.txt
+    part_config: /fsx-CoreModeling/zhanhouy/home/m5gnn_dataset/us_ads_sourcing/20230501_20230531-3clicks-002ctr-20maxa2a-enriched-100c2q-2c2aclicks-2c2apurchases-evalv5-debug-json-graph-dgl-graph/dist_2/20230501_20230531-3clicks-002ctr-20maxa2a-enriched-100c2q-2c2aclicks-2c2apurchases-evalv5-debug-json-graph.json
+    verbose: false
+    save_perf_results_path: null
+  gnn:
+    model_encoder_type: rgcn
+    fanout: "query/clicks/asin:20@asin/clicks-rev/query:20@asin/substitutes/asin:20@asin/substitutes-rev/asin:20@asin/a2b/brand:20@brand/a2b-rev/asin:20@asin/a2pt/pt:20@pt/a2pt-rev/asin:20"
```

Comment:
as we discussed, the distillation pipeline doesn't need to invoke GNN computation. why do we need these arguments?